src/ALZ/Private/Deploy-Accelerator-Helpers/Invoke-Terraform.ps1

```diff
@@ -89,7 +97,9 @@ function Invoke-Terraform {
             throw "Terraform plan failed with exit code $exitCode. Please review the error and try again or raise an issue."
         }
 
-        if(!$autoApprove) {
+        if($planOnly) {
+            return
+        } elseif(!$autoApprove) {

```

Comment:
Can we just make the a second if statement for readability since the two options are not directly related?
```suggestion
        } 
        if(!$autoApprove) {
```